.github/CONTRIBUTING.md

```diff
@@ -1,31 +1,32 @@
-# Contributing to KeePassX Reboot
+# Contributing to KeePassXC
 
 :+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
 
-The following is a set of guidelines for contributing to KeePassX Reboot on GitHub.
+The following is a set of guidelines for contributing to KeePassXC on GitHub.
 These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-#### Table Of Contents
+#### Table of contents
 
 [What should I know before I get started?](#what-should-i-know-before-i-get-started)
   * [Open Source Contribution Policy](#open-source-contribution-policy)
 
-[How Can I Contribute?](#how-can-i-contribute)
-  * [Feature Requests](#feature-requests)
-  * [Bug Reports](#bug-reports)
-  * [Your First Code Contribution](#your-first-code-contribution)
-  * [Pull Requests](#pull-requests)
+[How can I contribute?](#how-can-i-contribute)
+  * [Feature requests](#feature-requests)
+  * [Bug reports](#bug-reports)
+  * [Discuss with the team](#discuss-with-the-team)
+  * [Your first code contribution](#your-first-code-contribution)
+  * [Pull requests](#pull-requests)
   * [Translations](#translations)
 
 [Styleguides](#styleguides)
-  * [Git Branch Strategy](#git_branch_strategy)
-  * [Git Commit Messages](#git-commit-messages)
-  * [Coding Styleguide](#coding-styleguide)
+  * [Git branch strategy](#git-branch-strategy)
+  * [Git commit messages](#git-commit-messages)
+  * [Coding styleguide](#coding-styleguide)
 
 
 ## What should I know before I get started?
 ### Open Source Contribution Policy
-[Version 0.3, 2015–11–18](https://medium.com/@jmaynard/a-contribution-policy-for-open-source-that-works-bfc4600c9d83#.i9ntbhmad)
+[Version 0.3, 2015–11–18](https://medium.com/@jmaynard/a-contribution-policy-for-open-source-that-works-bfc4600c9d83#.i9ntbhmad) (*Copied below.*)
```

Comment:
The contents of that link (the policy) are copied and presented below in the CONTRIBUTING document – it seemed weird to me the first time, since I thought the link was provided for me to go read the external document, like with the branching system.